jboss-eap-cd-openshift-launch/added/standalone-openshift.xml

```diff
@@ -420,7 +420,9 @@
                     <protocol type="FRAG3"/>
                 </stack>
                 <stack name="tcp">
-                    <transport type="TCP" socket-binding="jgroups-tcp"/>
+                    <transport type="TCP" socket-binding="jgroups-tcp">
+                       <property name="use_ip_addrs">true</property>
```

Comment:
Ken,  our profiles are using 4 spaces; not 3.

Reply:
Ah, well spotted :)

Reply:
WS change is here: https://github.com/jboss-container-images/jboss-eap-modules/pull/6/files

@rcernich Can you update https://issues.jboss.org/browse/CLOUD-2425 to be in Sprint 16? I'll link the PRs once it's in. It seems weird to have  JBEAP driving this.